Pkgfile

```diff
@@ -173,9 +173,9 @@ vars:
   arm_trusted_firmware_sha512: 958cf8f9e258638e59d0fbd8b053fce7d8a9ea2fc922686c9d20ea16f79f55219ac18a12ab240c528ee98e49c2e0eef4c963fdb255cc14b92437a5b3cffc8640
 
   # renovate: datasource=github-tags extractVersion=^v(?<version>.*)$ depName=u-boot/u-boot
-  uboot_version: 2023.01
-  uboot_sha256: 69423bad380f89a0916636e89e6dcbd2e4512d584308d922d1039d1e4331950f
-  uboot_sha512: 417a28267eb7875820d08fafc7316f164663609378637539e71648b0b9b7d28796b6c381717f31b0ab6472805fefd32628ef7d1b2e7b9f3c51c8ad122993f679
+  uboot_version: 2023.10
```

Comment:
does this even build?

Reply:
It does for `linux/amd64` but looks like it doesn't for `linux/arm64`. 🤷 

```
2023-11-01T11:18:54.0247386Z  > linux/arm64 u-boot:prepare-0:
2023-11-01T11:18:54.0247792Z 3.954 patching file include/configs/rpi.h
2023-11-01T11:18:54.0248198Z 3.954 Hunk #1 FAILED at 131.
2023-11-01T11:18:54.0248527Z 3.954 Hunk #2 FAILED at 151.
2023-11-01T11:18:54.0249136Z 3.954 2 out of 2 hunks FAILED -- saving rejects to file include/configs/rpi.h.rej
2023-11-01T11:18:54.0249733Z ------
2023-11-01T11:18:54.0250671Z WARNING: No output specified with remote driver. Build result will only remain in the build cache. To push result image into registry use --push or to load image into docker use --load
2023-11-01T11:18:54.0255819Z ERROR: failed to solve: process "/toolchain/bin/bash -c set -eou pipefail\nmkdir ${RPI_ARM64_GENERIC_U_BOOT}\ntar -xjf u-boot.tar.bz2 --strip-components=1 -C ${RPI_ARM64_GENERIC_U_BOOT}\ncd ${RPI_ARM64_GENERIC_U_BOOT}\npatch -p1 < /pkg/rpi_arm64/0002-rpi-add-NVMe-to-boot-order.patch\npatch -p1 < /pkg/rpi_arm64/0003-Revert-nvme-Correct-the-prps-per-page-calculation-me.patch\npatch -p1 < /pkg/rpi_arm64/0004-usb-xhci-brcm-Make-driver-compatible-with-downstream.patch\npatch -p1 < /pkg/rpi_arm64/0005-nvme-improve-readability-of-nvme_setup_prps.patch\npatch -p1 < /pkg/rpi_arm64/0006-nvme-Use-pointer-for-CPU-addressed-buffers.patch\npatch -p1 < /pkg/rpi_arm64/0007-nvme-translate-virtual-addresses-into-the-bus-s-addr.patch\npatch -p1 < /pkg/rpi_arm64/0008-enable-nvme-and-fixup-efi-boot.patch\nmake rpi_arm64_defconfig\nsed -i \"s/CONFIG_TOOLS_LIBCRYPTO=y/# CONFIG_TOOLS_LIBCRYPTO is not set/\" .config\n" did not complete successfully: exit code: 1
```

Reply:
there's no u-boot for amd64, we should hold on updating this until we have time to figure out all patches

Reply:
Oh, okay. I tested locally with `PLATFORM=linux/amd64` so I didn't notice that.